.gitignore

```diff
@@ -19,3 +19,6 @@ pnpm-debug.log*
 
 # macOS-specific files
 .DS_Store
+
+# 11ty
+.cache
```

Comment:
This should be removed, because we got rid of the caching logic.

Reply:
Done!